packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -162,7 +163,7 @@ export class SummarizerNode implements IRootSummarizerNode {
 						summarySequenceNumber: this.wipReferenceSequenceNumber,
 						latestSummarySequenceNumber: this._latestSummary.referenceSequenceNumber,
 						// TODO: remove summaryPath
```

Comment:
I'm not a fan of `// TODO` comments in the code, especially if they don't have any details about why or when.  Can you either add more info here or just remove it?

Reply:
If you want to add the reason, it's so that we don't leak the id's of the parent datastore or parent runtime down to the datastore itself.